bleak/backends/bluezdbus/manager.py

```diff
@@ -284,44 +285,45 @@ async def async_init(self):
         connected, no action is performed.
         """
         async with self._bus_lock:
-            if self._bus.connected:
+            if self._bus and self._bus.connected:
                 return
 
-            await self._bus.connect()
+            bus = MessageBus(bus_type=BusType.SYSTEM)
```

Comment:
Can we get a comment here explaining why we need to create a new bus object instead of reusing the old one?

Reply:
Done in c581d5e76c936b86d9d460f0a5fc4ffcf84a199b